applications/visualizer/frontend/src/components/ViewerContainer/CustomListItem.tsx

```diff
@@ -55,7 +58,9 @@ const CustomListItem = ({
   };
 
   const handleColorChange = (color) => {
-    setSelectedColor(color.hex);
+    const hexColor = color.hex;
+    setSelectedColor(hexColor);
+    workspace.changeNeuronColorForViewers(data.id, hexColor, [ViewerType.ThreeD, ViewerType.EM]);
```

Comment:
The code you wrote is the main idea, but I think it might be better to have a callback and that this code would be relocated in the `Neuron` component. This way, we keep all the code related to workspace and workspace changes inside of the Neuron component instead of having it placed to a lower component like this one.

Also, following the signature of the function, it might not be necessary to pass the `[ViewerType., ...]` as argument of the method :)